MAINTAINERS.yml

```diff
@@ -1240,6 +1240,16 @@ Filesystems:
   labels:
     - "area: File System"
 
+"Filesystems: FatFs":
```

Comment:
Done.
But I am not quite sure if i should add me as a maintainer because I feel like I am just a collaborator. But I don't know if I can add a subsystem entry without a maintainer. Any ideas? @de-nordic @nordicjm